HISTORY.md

```diff
@@ -4,6 +4,7 @@
 ### New Features
 ### Bug Fixes
 * Fix IOError on WAL write doesn't propagate to write group follower
+* Fix iterator not become invalid on merge error
```

Comment:
How about changing this line as well as the PR title to something like "Make iterator invalid on merge error."

Reply:
That's more straight-forward.